public/kibi_timeline_vis_controller.js

```diff
@@ -70,6 +70,10 @@ define(function (require) {
           $scope.visOptions.groups = [];
           _.each(savedSearchesRes, function ({ savedSearch, groups }, i) {
             for (const group of groups) {
+              // get parent field if multifield
+              // because doc_values feature is not applied to string fields
+              group.labelField = group.labelField.length > 0 ? group.labelField.split('.')[0] : group.labelField;
```

Comment:
you should use `labelFieldSequence` instead, if available. In the searchsource, you have to add `source` and `fielddata_fields` https://www.elastic.co/guide/en/elasticsearch/reference/2.4/search-request-fielddata-fields.html
This way you can still get the value of the `raw` multifield in kibana.